students/dabeen/westagram/users/views.py

```diff
@@ -1,3 +1,77 @@
 from django.shortcuts import render
 
+import json
+import bcrypt
+import re
+import jwt
+from django.http import JsonResponse
+from django.views import View
+from users.models import User
+
 # Create your views here.
+
+
+class SignUpView(View):
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+
+            email = data['email']
+            password = data['password']
+
+            # Return 400 error when using an existing email.
+            if User.objects.filter(email=email).exists():
+                return JsonResponse({'message': 'EMAIL_ALREADY_EXISTS'}, status=400)
+
+            regex_email = '^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$'
+            regex_password = '^(?=.*[A-Z])(?=.*[a-z])(?=.*[0-9]).{8,}'
```

Comment:
정규식을 complie하지 않고 상수로 선언해서 사용하셨네요!  👍 👍 
추가로 상수는 대문자로 작성해주세요.

```suggestion
            REGEX_EMAIL = '^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$'
            REGEX_PASSWORD = '^(?=.*[A-Z])(?=.*[a-z])(?=.*[0-9]).{8,}'
```